polylogue/index_health.py

```
@@ -0,0 +1,65 @@
+from typing import List
+
+from .db import DB_PATH
+
+
+def verify_sqlite_indexes(db_path: Path = DB_PATH, *, attempt_rebuild: bool = True) -> List[str]:
+    issues: List[str] = []
+    conn = sqlite3.connect(db_path)
+    try:
+        status = conn.execute("PRAGMA integrity_check").fetchone()[0]
+        if status != "ok":
+            raise RuntimeError(f"SQLite integrity check failed: {status}")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)